src/Xamarin.Android.Build.Tasks/Tasks/CheckClientHandlerType.cs

```diff
@@ -16,23 +16,12 @@ public class CheckClientHandlerType : AndroidTask
 
 		[Required]
 		public string ClientHandlerType { get; set; }
-		[Required]
-		public string ValidHandlerType { get; set; }
+
 		[Required]
 		public ITaskItem[] ResolvedAssemblies { get; set; }
-		public bool UsingAndroidNETSdk { get; set; }
 
 		public override bool RunTask ()
 		{
-			// Fast path for known types
-			if (UsingAndroidNETSdk) {
-				if (ClientHandlerType == "Xamarin.Android.Net.AndroidMessageHandler")
-					return !Log.HasLoggedErrors;
-			} else {
-				if (ClientHandlerType == "Xamarin.Android.Net.AndroidClientHandler")
-					return !Log.HasLoggedErrors;
-			}
```

Comment:
I added these in d4c2a5e2ba148ea1d63fdc7f8c984642e7b5b369 to help performance. It should just be returning early for a valid, known value. Can we remove just the `else` block, assuming `UsingAndroidNETSdk` will never be false in .NET 6+?

Reply:
I tried to move this to the targets file as I assumed that would be even faster: https://github.com/xamarin/xamarin-android/pull/8438/files#diff-e2c60a31385c6fbadf9b9f439d3681ee357077bf3e66f181a0ddb02b9d98adf1R489

I can restore this in the task though if we prefer.